__test__/google-login-dist.nottest.js

```diff
@@ -0,0 +1,189 @@
+// import React from 'react'
```

Comment:
lets just delete these instead of renaming them

Reply:
removed file